clap_derive/tests/subcommands.rs

```diff
@@ -454,3 +454,24 @@ fn update_ext_subcommand() {
     opt.try_update_from(&["test", "ext", "42", "44"]).unwrap();
     assert_eq!(Opt::parse_from(&["test", "ext", "42", "44"]), opt);
 }
+
+#[test]
+fn subcommand_name_not_literal() {
+    fn get_name() -> String {
+        "renamed".to_string()
```

Comment:
Could you change this to returning `&'static str`?  If people use this as an example, it'd be good for them to go with more preferred case.